Pendulum/Extensions/UserDefaults.swift

```diff
@@ -56,6 +57,12 @@ extension UserDefaults {
         set { setValue(newValue, forKey: Key.shouldShowDebugView.rawValue) }
     }
     
+    @objc
```

Comment:
Why is this `@objc`?

Reply:
I don't see any usage that would require it for this, and probably for all others too (haven't checked all very well)